### PR TITLE
fix some typos and grammar on home page

### PR DESCRIPTION
**Pull request by zachvalenta**

### CODE REVIEW

**Comment by npetzall** [comment]
Hi, would you be willing to discuss the whole about section?
English isn't my first language and I'll admit when it comes to my first language I'm no poet.

**Comment by zachvalenta** [comment]
sure thing :)

**Comment by npetzall** [comment]
So the first paragraph in About is still published as:
>project is continue of work begin by John Currier in 2004 and improved during the years with greate community support.
You can find out more about initial project on site http://schemaspy.sourceforge.net

The second paragraph
>I would like continusly improving SchemaSpy and release new version of this greate tool becouse we don't have any release since version 5.0.0 released in 2010.
I'm personally believe that work on SchemaSpy should be continued and a lot of still existing issue should be resolved.
I would like to say GOOD Thank you to John Currier for invent this database entity-relationship (ER) diagram generator.

Think we can condense it to:

>This is the continuation of the great work started by John Currier in 2004, which can be found at [SourceForge](http://schemaspy.sourceforge.net), our goal is to release fixes, enhancements and new features.

But we are still missing the actual about SchemaSpy what it is. 

> SchemaSpy is a database analyzer using JDBC and custom queries to produce html report with entity-relationship diagrams, xml representation of the database. It can find implied relationships based on naming and data type. Currently it requires Java 8 to run, jdbc drivers to connect to database and [graphviz](http://www.graphviz.org) to create diagrams. Docker Image is available.

What do you think? Is the grammar ok, reading flow and content good?

**Comment by zachvalenta** [comment]
The explanation of what SchemaSpy is should be at the top of the about page. I like my edits for the background section.

**Comment by npetzall** [comment]
So something like:
### About
SchemaSpy is a database analyzer using JDBC and custom queries to produce html report with entity-relationship diagrams, xml representation of the database. It can find implied relationships based on naming and data type. Currently it requires Java 8 to run, jdbc drivers to connect to database and graphviz to create diagrams. Docker Image is available.

This project continues the work began by John Currier in 2004 and has improved over the years with great support from our community.  
You can find out more about the initial project on [Source Forge](http://schemaspy.sourceforge.net)

I would like to continuously improve SchemaSpy and to release a new version of this great tool because we haven't had any releases since version 5.0.0 was released in 2010.  
I'm personally believe that work on SchemaSpy should be continued and a lot of the still-existing issues should be resolved.  
I would like to say a BIG thank you to John Currier for inventing this database entity-relationship (ER) diagram generator.

File: index.html [hunk 1]
```diff
@@ -179,12 +179,12 @@ <h3 class="section-subheading text-muted">Here you will find more detials about
...
+                    <h3 class="section-subheading text-muted">This project continues the work began by John Currier in 2004 and has improved over the years with great support from our community.</br>
+                    You can find out more about the initial project on <a href="http://schemaspy.sourceforge.net">Source Forge</a>
                     </h3>
-                    <p>I would like continusly improving SchemaSpy and release new version of this greate tool becouse we don't have any release since version 5.0.0 released in 2010.</br>
-                    I'm personally believe that work on SchemaSpy should be continued and a lot of still existing issue should be resolved.</br>
-                    I would like to say GOOD Thank you to John Currier for invent this database entity-relationship (ER) diagram generator.</p>                      
+                    <p>I would like to continuously improve SchemaSpy and to release a new version of this great tool becouse we haven't had any releases since version 5.0.0 was released in 2010.</br>
```
Comment (npetzall) [inline index.html]: becouse = because

Comment (zachvalenta) [reply index.html]: yep!

only thing: "I'm personally believe" --> "I personally believe"

**Comment by zachvalenta** [comment]
Can we get this merged?

**Pull request merged**